azure_arc_k8s_jumpstart/aks_stack_hci/gitops/basic/az_k8sconfig_aks_hci.ps1

```diff
@@ -0,0 +1,44 @@
+# <--- Change the following environment variables according to your Azure service principal name --->
+
+Write-Host "Exporting environment variables"
+$appId ='<Your Azure service principal name>'
+$password ='<Your Azure service principal password>'
+$tenant ='<Your Azure tenant ID>'
+$resourceGroup ='<Azure resource group name>'
+$ClusterName ='<The name of your AKS cluster running on Azure Stack HCI>'
+$appClonedRepo ='<The URL for the "Hello Arc" cloned GitHub repository>'
+$subscriptionId ='<Your subscription ID>'
+
+
+# Connect to Azure
+Write-Host "Log in to Azure with Service Principal & Getting AKS credentials (kubeconfig)"
+az login --service-principal --username $appId --password $password --tenant $tenant
+az account set --subscription $subscriptionId
+
+#Get AKS on Azure Stack HCI cluster credentials
+Get-AksHciCredential -Name $ClusterName 
+
+# Create a namespace for your ingress resources
+kubectl create namespace hello-arc
+
+# Helm Install 
+
```

Comment:
```suggestion

```

Reply:
I am missing the automation for the k8s cli extensions (connectedk8s and k8s-configuration)

Reply:
I cannot do the install from the PowerShell script I will include in the prerequisites